x-pack/docs/en/ilm/apis/retry-policy.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-put-policy]]
+=== Retry Policy Execution API
+++++
+<titleabbrev>Retry Policy Execution</titleabbrev>
+++++
+
+Retry executing the policy for an index which has errored.
+
+==== Request
+
+`POST <index>/_ilm/retry`
+
+==== Description
+
+This API will re-run a policy is currently in the ERROR step. It will set the
```

Comment:
Perhaps link to the "get status" API so a user knows how to check if an index is in the error step